sw/ground_segment/python/udp_link/pprzlink_proxy.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import time
+
+# if PAPARAZZI_SRC not set, then assume the tree containing this
+# file is a reasonable substitute
+PAPARAZZI_HOME = os.getenv("PAPARAZZI_HOME", os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)),'../../../')))
+sys.path.append(PAPARAZZI_HOME + "/var/lib/python")
+
+import pprzlink.udp
+
+#DEFAULT_ADDRESS= "127.255.255.255"
+DEFAULT_ADDRESS= "127.0.0.1"
+
+# GCS ID, OUT port, IN port
+gcs_conf = [0, 4243, 4242]
+
+# list ID, OUT port, IN port
+mavs_conf = [
+        [102, 4244, 4245],
+        [103, 4246, 4247],
+        [104, 4248, 4249]
+        ]

```

Comment:
One would expect these to be optional program inputs (which could default to these values if not defined). Same with the DEFAULT_ADDRESS and the gcs_conf.

Reply:
Totally agree. In fact, Gautier has plans for this file. 

This was a quick patch for making the simulation work with air-to-air messages and it should not be here indeed.